### DISCUSSION TITLE
Device Aurora Smart16ARelay51AU

### DISCUSSION THREAD

**Discussion by poirs01** [Koenkk/zigbee2mqtt#14668] [Devices]
Hello,
I installed an Aurora Smart16ARelay51AU light control, but it is not supported.
How can I use it and make it compatible?
Thanks
Ju

**Comment by Koenkk** [comment]
To request support for a new device fill in https://github.com/Koenkk/zigbee2mqtt/issues/new?assignees=&labels=new+device+support&template=new_device_support.yaml&title=%5BNew+device+support%5D%3A+
